stdlib/ast.pyi

```diff
@@ -1868,7 +1885,7 @@ else:
         *,
         type_comments: bool = False,
         feature_version: None | int | tuple[int, int] = None,
-    ) -> AST: ...
+    ) -> mod: ...
```

Comment:
This is incorrect, it's for the case where `mode` is not a known string, so it may return some other kind of AST.

Reply:
Oh wait, `mod` is a superclass of all the things this can return, so yes this is correct.